src/App.js

```diff
@@ -1,11 +1,78 @@
+import ContestsPage from './PagesBody/ContestsPage/ContestsPage';
+import DogDataPage from './PagesBody/DogDataPage/DogDataPage';
+import DogSummaryPage from './PagesBody/DogSummaryPage/DogSummaryPage';
+import ErrorTestPage from './PagesBody/ErrorTestPage/ErrorTestPage';
+import ExercisesPage from './PagesBody/ExercisesPage/ExercisesPage';
+import ForgotPassForm from './Organisms/ForgotPassForm/ForgotPassForm';
+import LandingPage from './PagesBody/LandingPage/LandingPage';
+import Layout from './Templates/Layout/Layout';
+import LeaderboardPage from './PagesBody/LeaderboardPage/LeaderboardPage';
+import LoginForm from './Organisms/LoginForm/LoginForm';
+import ModalsTest from './PagesBody/ModalsTest';
+import NotFoundPage from './PagesBody/NotFoundPage/NotFoundPage';
+import ParticipantDataPage from './PagesBody/ParticipantDataPage/ParticipantDataPage';
+import ProfilePage from './PagesBody/ProfilePage/ProfilePage';
+import RegistrationForm from './Organisms/RegistrationForm/RegistrationForm';
+import RolePage from './PagesBody/RolePage/RolePage';
+import UnregisteredPage from './PagesBody/UnregisteredPage/UnregisteredPage';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header"></header>
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route element={<Layout withSettings />}>
+          <Route path="user" element={<ProfilePage />} />
+        </Route>
+        <Route element={<Layout />}>
+          <Route path="testErrors" element={<ErrorTestPage />} />
+          <Route path="*" element={<NotFoundPage />} />
+          <Route path="forgot" element={<ForgotPassForm />} />
+          <Route path="contactForm" element={<ContactFormPage />} />
+          <Route path="register" element={<RegistrationForm />} />
+          <Route path="unRegistered" element={<UnregisteredPage />} />
+          <Route path="login" element={<LoginForm />} />
+        </Route>
+        <Route element={<Layout withLabel />}>
+          <Route path="role" element={<RolePage />} />
+          <Route path="contests" element={<ContestsPage />} />
+          <Route path="contests/:contestId/classes" element={<ClassesPage />} />
+          <Route
+            path="contests/:contestId/classes/:classId"
+            element={<ClassCompetitorsPage />}
+          />
+          <Route
+            path="contests/:contestId/classes/:classId/leaderboard"
+            element={<LeaderboardPage />}
+          />
+          <Route
+            path="contests/:contestId/classes/:classId/:dogId"
+            element={<ExercisesPage />}
+          />
+          <Route
+            path="contests/:contestId/classes/:classId/:dogId/dogSummary"
+            element={<DogSummaryPage />}
+          />
+          <Route path="dogData" element={<DogDataPage />} />
+          <Route path="participantData" element={<ParticipantDataPage />} />
+          <Route path="contestDetails" element={<ContestDetailsPage />} />
```

Comment:
Try to make all paths case insensitive, kebab-case or snake_case

Reply:
fixed in #203